src/main/java/com/gmail/woosay333/onlinebookstore/controller/BookController.java

```diff
@@ -19,17 +21,30 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
```

Comment:
I can not leave a comment that far up but this is about the general `@RequestMapping("/books")` which should actually be `/api/books` as per the requirements in the previous task I think